openbb_platform/providers/intrinio/openbb_intrinio/models/cash_flow.py

```diff
@@ -93,16 +99,21 @@ def extract_data(
         url_params = f"{query.symbol}-cash_flow_statement-{query.year}"
         statement_param = f"{query.type}_financials"
 
-        data = []
+        data: List[Dict] = []
 
         if query.period == "annual":
             url = f"{base_url}/fundamentals/{url_params}-FY/{statement_param}?api_key={api_key}"
             data.append(get_data_one(url, **kwargs))
 
         elif query.period == "quarter":
-            # TODO: Fix quarter range after Intrinio's response
-            for quarter in range(1, 4):
-                url = f"{base_url}/fundamentals/{url_params}-Q{quarter}/{statement_param}?api_key={api_key}"
+            quarter_range = get_quarter_range(query.year)
+
+            for quarter in quarter_range:
+                # TODO: Check back in sometime when Intrinio fixes their API/provides better documentation
+                if quarter == 4 and query.type == "reported":
```

Comment:
This does not appear to work:

![Screenshot 2023-10-19 at 12 24 00 PM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/085f0b80-09bd-467e-9457-4e2dc5cfeb8d)


Changing the symbol to "JPM" creates an error:

```
OpenBBError: Intrinio endpoint doesn't exist
```

Tickers that do return, return all NaNs for Q2 and Q3

![Screenshot 2023-10-19 at 12 28 59 PM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/df9f72f3-3e53-41b3-89f5-75245a149797)